package.json

```diff
@@ -27,6 +27,7 @@
   "devDependencies": {
     "@rescript/react": "^0.10.1",
     "bs-platform": "^8.3.0",
+    "parcel-bundler": "^1.12.5",
```

Comment:
Maybe add a `dev` command in `scripts`  to run the examples

Reply:
Yeah, I have it, but forgot to commit. Will do